src/renate/utils/deepspeed.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2020 The PyTorch Lightning team and Microsoft Corporation. All rights reserved.
```

Comment:
what is with this copyright note?

Reply:
Its an adaptation of lightning utils file. So this has an original copyright, and modifications are according to our spec. See line 45. Should this be changed?